constants.py

```diff
@@ -0,0 +1,13 @@
+
+# These are used to figure out what amount something needs
+# to be scaled when importing or exporting.
+SCALE_MULTIPLIERS = {
+	# 10       : A world unit is 10 feet.
+	# 0.032808 : feet per centimeter
```

Comment:
There's a lossless way to do this conversion if you flip the math and do centimeters per feet instead.
2.54 cm -> 1in is an exact conversion so 30.48 cm -> 1 ft is as well.